### PR TITLE
Ask the user if they want to force a non-upgrade

### LINKED ISSUE #16: Add an interactive prompt to the update command when downgrading
Currently, we require users to use the `--downgrade` (soon to be `--force`) flag, which forces them into non-interactive mode. This is unideal since downgrading is usually going to be done interactively.

### PR DESCRIPTION
**Pull request by velovix**
Prompts the user asking if they want to force a non-upgrade if they're in interactive mode, instead of requiring the `--force` flag.

Fixes #16.

### CODE REVIEW

**Review by apockill — APPROVED** [review]

**Comment by BryceBeagle** [comment]
Is "non-upgrade" a real term? Why not downgrade?

**Comment by velovix** [comment]
> Is "non-upgrade" a real term? Why not downgrade?

I used this term to cover both a downgrade and an attempt to "upgrade" to the version that's already installed.

**Review by BestDriverCN — APPROVED** [review]

**Comment by apockill** [comment]
@velovix This seems ready to merge?

**Comment by BryceBeagle** [comment]
I'm still against the "non-upgrade" terminology. I think it would be better to make it clear. Non-upgrade certainly won't translate well

**Comment by velovix** [comment]
Yeah, and I haven't had a chance to address that yet since this PR is more of a side project. I might get to it this weekend if I feel like it 😏

**Comment by velovix** [comment]
@BryceBeagle Is this okay?

File: brainframe/cli/commands/update.py [hunk 1]
```diff
@@ -25,7 +25,19 @@ def update():
     existing_version = version.parse(existing_version_str)
     requested_version = version.parse(requested_version_str)
 
-    if not args.force:
+    force_downgrade = False
+    if args.noninteractive:
+        # Use the --force flag to decide if downgrades are allowed
+        force_downgrade = args.force
+    else:
```
Comment (BryceBeagle) [inline brainframe/cli/commands/update.py:32]: Why would the `--force` flag not work when interactive?

Comment (velovix) [reply brainframe/cli/commands/update.py:32]: Because the CLI goes into non-interactive mode if any flags are provided.

**Review by BryceBeagle — APPROVED** [review]

**Pull request merged**